modules/user_accounts/templates/menu_user_accounts.tpl

```diff
@@ -2,7 +2,7 @@
 
 <!-- start the selection table -->
 <div class="row">
-    <div class="col-sm-9 col-lg-8">
+    <div class="col-sm-9 col-lg-12">
```

Comment:
I think the 1/3 block of white space we have currently is a bit weird for sure. I don't see us putting anything else in that space, although we may still have something in candidate list where you can enter the PSCID/DCCID to open a candidate? I can't remember if we removed that or not.

I prefer the full width idea for now. And hopefully we can further revamp filters in the near future.